src/remote/RemoteComponent.cpp

```diff
@@ -85,6 +85,38 @@ QVariantMap RemoteComponent::ResourceInformation()
   resourceInfo["platform"] = QSysInfo::productType();
   resourceInfo["platformVersion"] = QSysInfo::productVersion();
 
+  // Vary the `title` value depending on the `systemname` value in
+  // plexmediaplayer.conf. The `title` is used by the UI when publishing PMP as
+  // a Plex Companion player.
+  //
+  // The `gdmInfo` contains a `title` value which is equivalent to the system's
+  // hostname. The OESystemComponent then sets the `systemname` as system's
+  // hostname. Whereas the `systemname` property is ignored on non OE systems.
+  // So, on OE systems the resulting `title` value is always the value of the
+  // `systemname` property while the `title` on desktop systems is always the
+  // system's actual hostname.
+  //
+  // Always return a non-default value for `systemname`. The default value
+  // `systemname` value is always ignored in favor of a the more descriptive
+  // system hostname provided by GDM. The default value for `systemname` _and_
+  // system hostname from GDM are equivalent on OE which can lead to a
+  // pathological case of an empty `systemname`.
+  //
+  // See: https://github.com/plexinc/plex-media-player/issues/762
+  QString systemname = SettingsComponent::Get().value(SETTINGS_SECTION_SYSTEM, "systemname").toString();
+  QString defaultSystemname = "PlexMediaPlayer";
```

Comment:
I dislike this hardcoded string but I've been unable to find a suitable application constant.

Reply:
Wouldn't we prefer a friendlier `Plex Media Player`?

Reply:
Coming back to this PR a bit late. In general absolutely. I'm hesitant to change as the value also becomes the host system's name on embedded systems. Just trying to work within the box. 🤕